### PR TITLE
ui: Allow IP range creation for Physical Network - Guest Traffic in Basic Zones

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

For Basic Zones, additional IP ranges can be added for the guest traffic, however, in the new UI, it isn't possible to add IP ranges.
Legacy UI:
image

Current UI:
Doesn't have the provision for adding additional IP range.
image

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [X] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
image

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan ui

File: ui/src/views/infra/network/TrafficTypesTab.vue [hunk 1]
```diff
@@ -129,6 +140,35 @@ export default {
...
+        const zone = json.listzonesresponse.zone || []
+        this.networkType = zone[0].networktype
+      }).catch(error => {
+        this.$notifyError(error)
+      }).finally(() => {
+        this.fetchLoading = false
+      })
+    },
+    fetchGuestNetwork () {
+      if (this.networkType === 'Basic') {
```
Comment (shwstppr) [inline ui/src/views/infra/network/TrafficTypesTab.vue:155]: @Pearl1594 Can you please check if this even called. Default value for `networkType` is `Advanced` and `fetchGuestNetwork` is called before finding `networkType` at line 147. I think `network` object of IpRangesTabPublic.vue will always remain empty.

**Review by shwstppr — CHANGES_REQUESTED** [review]

**Comment by rhtyd** [comment]
Is it available if you go to the network -> ip ranges? I think I saw some change by someone recently for shared networks (adding IP range).

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Comment by Pearl1594** [comment]
@rhtyd I don't see that option 
image

**Comment by shwstppr** [comment]
@Pearl1594 it is there in 4.15 branch I think
Screenshot from 2021-05-12 17-34-02

Though I'm fine we want to provide it in Infra section as well.
cc @rhtyd

**Review by utchoang — APPROVED** [review]
LGTM!
A few things in the network area, such as the @rhtyd comment, need to be tweaked.
cc @shwstppr

**Review by shwstppr — APPROVED** [review]
LGTM. Tested, it works fine.
Will be good to have for the consistency in the UI as compared to legacy UI. People accustomed to legacy UI might still use this workflow.

**Pull request merged**